modules/hivemq-edge-module-opcua/src/main/java/com/hivemq/edge/adapters/opcua/OpcUaProtocolAdapter.java

```diff
@@ -422,6 +424,7 @@ public void onSessionActive(final @NotNull UaSession session) {
             final BrowseResult browseResult = client.browse(browse).get();
```

Comment:
actually why are we even calling .get() here, the whole method is async, so why not chaining the futures at this point? This would result in removal of the catch of the Interrupt.

Reply:
The content of handleBrowseResult also doesn't look right. Gonna address this.